### PR TITLE
CORE-1102 Add 0355 Error code

### PR DESCRIPTION
**Pull request by viccomx**
0900 error indicates that no lookup was found with the given
hash.

Signed-off-by: Victor Cruz <victorcruz@Bitsos-MacBook-Pro.local>

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -177,6 +177,9 @@ error categories, the last two digits define specific errors.
 * 0801: You have hit the request rate-limit
 * 0802: Too many attempts to perform an operation
 
+### Other Errors: 09 (HTTP 420)
+* 0900: Not found users lookup hash
```
Comment (CrispinFlowerday) [inline source/index.html.md]: Why would this just not be a 04xx (validation error) ?

Comment (viccomx) [reply source/index.html.md]: I think it is not a validation error, they are trying to retrieve something that is not there, it is more like a business logic error I think.

Comment (CrispinFlowerday) [reply source/index.html.md]: Its the same sort of thing as `Incorrect WID (non-existent or does not belong to user)` though ... they asked for something that isn't there...

Comment (viccomx) [reply source/index.html.md]: Fixed

**Pull request merged**